pages/api/payment-fraud/place-order.js

```diff
@@ -0,0 +1,160 @@
+import { Sequelize } from 'sequelize';
+import {
+  areVisitorIdAndRequestIdValid,
+  messageSeverity,
+  CheckResult,
+  checkResultType,
+  getOkReponse,
+  getForbiddenReponse,
+  reportSuspiciousActivity,
+  getVisitorData,
+  checkFreshIdentificationRequest,
+  checkConfidenceScore,
+  checkIpAddressIntegrity,
+  checkOriginsIntegrity,
+} from '../../../shared/server';
+
+// Provision the database.
+// In the Stackblitz environment, this db is stored locally in your browser.
+// On the deployed demo, db is cleaned after each deployment.
+const sequelize = new Sequelize('database', '', '', {
+  dialect: 'sqlite',
+  storage: '.data/database.sqlite',
+  logging: false,
+});
+
+// Defines db model for login attempt.
+const PaymentAttempt = sequelize.define('payment-attempt', {
+  visitorId: {
+    type: Sequelize.STRING,
+  },
+  isChargebacked: {
+    type: Sequelize.BOOLEAN,
+  },
+  checkResult: {
+    type: Sequelize.STRING,
+  },
+  timestamp: {
+    type: Sequelize.DATE,
+  },
+});
+
+PaymentAttempt.sync({ force: false });
+
+const mockedCard = {
```

Comment:
Can't find usage of this

Reply:
Yep, I decided to remove it eventually, thank you, will remove it.